modules/dedicated-creating-your-cluster.adoc

```diff
@@ -7,7 +7,7 @@
 
 To create your {product-title} cluster:
 
-. Log in to link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift].
+. Log in to *https://cloud.redhat.com/openshift*.
```

Comment:
I don't know if we have it documented, but I do think it's a recommended practice to use the `link:` prefix for external links. One reason I can think of is that if you want an easy way to find links (for example, for broken link checking, etc.) the `link:` is an easy way to find them.

If you look at other places that link to cloud.redhat.com/openshift in our docs, they do use `link:`, and apparently there's an attribute of `cloud-redhat-com` that can be used for the title of it (Red Hat OpenShift Cluster Manager).